frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -176,9 +176,9 @@ class BitBox02 extends Component<Props, State> {
         const { sidebarStatus } = panelStore.state;
         apiGet(this.apiPrefix() + '/status').then(status => {
             const restoreSidebar = status === 'initialized' && !['createWallet', 'restoreBackup'].includes(appStatus) && sidebarStatus !== '';
-            if (restoreSidebar || status === 'connected') {
+            if (restoreSidebar) {
                 setSidebarStatus('');
-            } else if (status !== 'initialized' && ['', 'forceCollapsed'].includes(sidebarStatus)) {
+            } else if (status === 'connected' || (status !== 'initialized' && ['', 'forceCollapsed'].includes(sidebarStatus))) {
```

Comment:
I think this is redundant: `status !== 'initialized` includes `status === 'connected'`. Less complicated `if` statement -> easier to read.

Reply:
Indeed, will remove that and keep it as it was with `status !== 'initialized`

Reply:
fixed into the first commit https://github.com/digitalbitbox/bitbox-wallet-app/compare/32de3278444ebb540c3fc0ec003ba834f29f82e6..49856c1a2b43b623e72781cada59848194f1941d

Reply:
but it's still here in this commit. just remove it from 6410ff62b616674b123c78369d541fa3ada0d55c

Reply:
![image](https://user-images.githubusercontent.com/25405/127471025-9f626e24-168f-4bc5-802a-3c39558ca465.png)